### PR TITLE
add-htmlcssfile

### PR DESCRIPTION
**Pull request by josuerushanika**
In building our portfolio website, we created the mobile website section where we listed our portfolio projects.
In our Project, we did Portfolio by using templates in Figma HTML and CSS Properties like Flexbox; font, colors, images, text, and margins Implement the button interactions (enable, hover, etc.); We didn't implement any functionality that requires JavaScript, like opening the menu or creating dropdowns.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+         <ul class="card-tags">
+            <li>html</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+         </ul>
+         <button class="primary-button card-button" type="button">
+            see Project
+         </button>
+        </div>
+    </article>
```
Comment (abdulhamiid) [inline index.html]: Your design 👇 
image

Figma design 👇 
image

- [ ] Please kindly use the same font as [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in this section as mentioned in the[ project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @josuerushanika 👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @josuerushanika 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. 🙏
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    <section class="content">
+        <h1 class="intro">I'm Josue Rushanika Nice to see you!</h1>
+        <p class="intro1">
+            I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="intro2">LET'S CONNECT</h3>
+        <ul class="social">
+            <li class="social-icon twitter">
```
Comment (cindykandie) [inline index.html:48]: Good use of HTML Semantics Keep up😎

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (huzaifakhan0308) [inline .github/workflows/linters.yml:56]: I think all the things were already good so there is no need for more changes💥💫▶️▶️😎

**Pull request merged**